P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium.
+Dual-licensed under CC-by and BSD2 licences
+See the file COPYING.txt for details.
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="post" mode="add" module="cmc">
+    <desc xml:lang="en" versionDate="2022-08-09"> a written (or spoken) contribution to an ongoing
+        CMC interaction which has been composed (or recorded) by its author in its entirety as part
+        of a private activity, and has been transmitted through the internet and is made available
```

Comment:
Why is it a _private_ activity?

Reply:
2024-04-27 CMC and Council discussion: "private" is about the process of composition. We think we can revise this by simplifying the language: Simple rephrase of "as part of a private activity, and" to "before it has been transmitted via the internet"

Reply:
Re-worded in 759c2adba. Take a look and see if it is OK now.